components/admin_console/system_users/system_users_dropdown.jsx

```diff
@@ -612,7 +583,6 @@ export default class SystemUsersDropdown extends React.Component {
                 {makeDemoteModal}
                 {deactivateMemberModal}
                 {revokeSessionsModal}
-                {serverError}
```

Comment:
Instead of being displayed, `serverError` is being passed up into the parent component via the `onError(...)` method that was passed into this component's props